srcpkgs/espanso/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'espanso'
+pkgname=espanso
+version=2.1.6beta
```

Comment:
we should use a stable version

Reply:
Well, they have only released alpha or beta versions for years. See https://github.com/espanso/espanso/tags. The last one without the "beta" is probably obsolete. As a remark, Arch is also currently using the version 2.1.6. Without beta.